xbmc/guilib/GUIFontTTFGL.cpp

```diff
@@ -280,6 +282,9 @@ void CGUIFontTTFGL::LastEnd()
 
 CVertexBuffer CGUIFontTTFGL::CreateVertexBuffer(const std::vector<SVertex> &vertices) const
 {
+  assert(!vertices.empty());
```

Comment:
kodi-x11: /home/xbmc/progs/src/kodi/xbmc/guilib/GUIFontTTFGL.cpp:285: virtual CVertexBuffer CGUIFontTTFGL::CreateVertexBuffer(const std::vector<SVertex>&) const: Assertion `!vertices.empty()' failed.
Aborted (core dumped)

reproducible here by changing a timer on the backend during tv playback

Reply:
@FernetMenta Thanks, I'll have a look. Now I'm glad I added the assert even though I thought everything should be fine...